internal/guest/storage/scsi/scsi.go

```diff
@@ -114,7 +128,7 @@ func Mount(ctx context.Context, controller, lun uint8, target string, readonly b
 // Unmount unmounts a SCSI device mounted at `target`.
 //
 // If `encrypted` is true, it removes all its associated dm-crypto state.
-func Unmount(ctx context.Context, controller, lun uint8, target string, encrypted bool) (err error) {
+func Unmount(ctx context.Context, controller, lun uint8, target string, encrypted bool, verityInfo *prot.DeviceVerityInfo, securityPolicy securitypolicy.SecurityPolicyEnforcer) (err error) {
```

Comment:
Nothing needed right now with the added parameters?

Reply:
I think the dm-verity unmount calls are missing here.

Reply:
Doing in a followup PR.